src/chapters/data-manipulation/data-manipulation.rst

```diff
@@ -106,23 +104,27 @@ Check Your Understanding
 
 .. admonition:: Question
 
-   The default merge in the pandas merge() function is a left merge.
+   The default merge in the ``pandas`` ``merge()`` function is a left merge.
 
    a. True
    b. False
 
 .. admonition:: Question
 
-   Which merge combines ALL of the rows of the merged dataframes, filling in NaN if values are missing?
+   Which merge combines ALL of the rows of the merged DataFrames, filling in ``NaN`` if values are missing?
 
 .. admonition:: Question
-
-   (1 of 2) In the merge() function, there are the following parameters: on, left_on, and right_on. 
+   
+   ``merge()`` **question 1 of 2**
```

Comment:
suggestion to delte this line.

Reply:
Rewrote the question to help with clarity.